### PR TITLE
Fix copy resources

### PR DESCRIPTION
**Pull request by thinkeng**
CopyResources   add  convertCRIImage

### CODE REVIEW

File: keadm/cmd/keadm/app/cmd/util/image.go [hunk 1]
```diff
@@ -115,6 +115,7 @@ func (runtime *CRIRuntime) PullImage(image string, authConfig *runtimeapi.AuthCo
 // CopyResources copies binary and configuration file from the image to the host.
 // The same way as func (runtime *DockerRuntime) CopyResources
 func (runtime *CRIRuntime) CopyResources(edgeImage string, files map[string]string) error {
+	edgeImage = convertCRIImage(edgeImage)
```
Comment (Shelley-BaoYue) [inline keadm/cmd/keadm/app/cmd/util/image.go:118]: Why need to add this？  Is there any error occured?

**Comment by Shelley-BaoYue** [comment]
`DCO` need to be fixed. 😄

File: keadm/cmd/keadm/app/cmd/util/image.go (hunk 1 above)
Comment (thinkeng) [reply keadm/cmd/keadm/app/cmd/util/image.go:118]: 当镜像重命名或者不使用docker 镜像仓库的镜像，二十使用私有仓库镜像是，会报镜像无法加载的错误

Comment (thinkeng) [reply keadm/cmd/keadm/app/cmd/util/image.go:118]: `1201 21:35:36.352097 935665 join.go:184] 5. Copy resources from the image to the management directory
E1201 21:35:38.461963 935665 remote_runtime.go:176] "RunPodSandbox from runtime service failed" err="rpc error: code = Unknown desc = failed to setup network for sandbox \"f9713b839351af9c989aa02b36aba4152db30f0034041f40a64352744976e001\": plugin type=\"bridge\" failed (add): netplugin failed with no error message: signal: segmentation fault"
Error: edge node join failed: copy resources failed: rpc error: code = Unknown desc = failed to setup network for sandbox "f9713b839351af9c989aa02b36aba4152db30f0034041f40a64352744976e001": plugin type="bridge" failed (add): netplugin failed with no error message: signal: segmentation fault
execute keadm command failed: edge node join failed: copy resources failed: rpc error: code = Unknown desc = failed to setup network for sandbox "f9713b839351af9c989aa02b36aba4152db30f0034041f40a64352744976e001": plugin type="bridge" failed (add): netplugin failed with no error message: signal: segmentation fault`

Comment (Shelley-BaoYue) [reply keadm/cmd/keadm/app/cmd/util/image.go:118]: Thanks! Please solve the `DCO` error, check the `details` to get more details 
image

**Comment by Shelley-BaoYue** [comment]
There is an unrelated commit in this PR.

**Pull request closed without merging**